### PR TITLE
manpage: Improve description of $releasever and --releasever usecases…

### PR DESCRIPTION
**Pull request by mlandres**
… [bsc#1149511](https://bugzilla.opensuse.org/show_bug.cgi?id=1149511)

### CODE REVIEW

File: doc/zypper.8.txt [hunk 1]
```diff
@@ -656,7 +656,11 @@ include::{incdir}/option_Solver_Flags_Installs.txt[]
...
+	For all repositories which have the distribution version within their URL (like https://download.opensuse.org/distribution/'13.1'/repo/oss) using the *$releasever* variable instead may be helpful (https://download.opensuse.org/distribution/*$releasever*/repo/oss). The variable is per default substituted by the current distributions version ('13.1').
+	+
+	This value can be temporarily overwritten in the current zypper command by using the *--releasever* global option. Calling *zypper --releasever 13.2*'...' will cause these repos to use the new location (https://download.opensuse.org/distribution/'13.2'/repo/oss) without need to add/remove anything. But you'll need to use *--releasever 13.2* with every zypper command until the distribution upgrade was actually performed. Once the *dup* is done, *$releasever* will default to the new distribution version '13.2' and *--releasever* is no longer needed.
```
Comment (bzeller) [inline doc/zypper.8.txt]: without need to add/remove anything. -> without **the** need to add/remove anything.

File: doc/zypper.8.txt [hunk 2]
```diff
@@ -1713,7 +1727,7 @@ Repository Options: :: {nop}
 	Ignore remote repositories like http, ftp, smb and similar. This makes using zypper easier when being offline. When this option is specified, zypper acts as if the remote repositories were not defined at all.
 
 *--releasever* 'version'::
-	Set the value of the *$releasever* variable in all *.repo* files (default: 'current distribution version'). This can be used to switch to new distribution repositories when performing a distribution upgrade. See section *Repository Management* and the *dist-upgrade* (*dup*) command for details.
+	For the current command set the value of the *$releasever* repository variable to 'version'. This can be used to switch to new distribution repositories when performing a distribution upgrade. See the *dist-upgrade* (*dup*) command and section *Repository Management* for nore details about using the *$releasever* repository variable.
```
Comment (bzeller) [inline doc/zypper.8.txt]: nore ->  more

File: doc/zypper.8.txt (hunk 1 above)
Comment (mlandres) [reply doc/zypper.8.txt]: fixed

File: doc/zypper.8.txt (hunk 2 above)
Comment (mlandres) [reply doc/zypper.8.txt]: fixed

**Pull request merged**